### PR TITLE
Allowing multiple command line options

### PR DESCRIPTION
**Pull request by rosstroha**
Hi! Thanks for writing this karma plugin! I am planning on using it in another project but there is some missing functionality that I need. 

This PR is to allow multiple command line options to be given to the child process without breaking changes to current users of the plugin. 

Previously, when giving multiple options in the karma configuration file like this
`slimrbOption: '-l { "foosball": true } -r slim/smart'

... the child process would be given an array of options formatted like this:
```javascript
[
'-l { "foosball": true } -r slim/smart',
'some/file.slim'
]
```

Instead, it will simply accept and array of options and be given an array formatted like this:
```javascript
[
'-l { "foosball": true }', 
'-r slim/smart',
'some/file.slim'
]
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ function createSlimPreprocessor(config, logger) {
   if(typeof(config.slimrbOption) === 'string') {
     log.debug('Passing args to slim command (%s).', config.slimrbOption);
     slimCommandArgs.push(config.slimrbOption);
+  } else if(config && Array.isArray(config.slimrbOption)){
+    slimCommandArgs = config.slimrbOption;
```
Comment (ratbeard) [inline index.js]: Can you add a debug logger here, like above?

`log.debug('Passing args to slim command (%s).', config.slimrbOption);
`

**Review by ratbeard — CHANGES_REQUESTED** [review]

**Comment by ratbeard** [comment]
Thanks for the PR @rosstroha.  I don't have a project setup using this plugin anymore, so I couldn't test, but does this form not work?

```
[
'-l { "foosball": true } -r slim/smart',
'some/file.slim'
]
```

It doesn't look like a breaking change to me, as the old string style is still handled.

**Comment by rosstroha** [comment]
@ratbeard It doesn't seem to like it as a single string

When the options is set like this: `slimrbOption: '--trace -r slim/smart'`

The error returned is `... slim-3.0.8/lib/slim/command.rb:18:in `run': invalid option: --trace -r slim/smart (OptionParser::InvalidOption)`

**Comment by rosstroha** [comment]
> It doesn't look like a breaking change to me, as the old string style is still handled.

Woops, I totally forgot a word when writing the PR. My bad! I've edited it.

File: index.js (hunk 1 above)
Comment (ratbeard) [reply index.js]: ping

Comment (rosstroha) [reply index.js]: Sorry about that. Pushed!

**Comment by ratbeard** [comment]
No problem.  Looks good, thanks for contributing 🏖

**Pull request merged**